composer.json

```diff
@@ -20,6 +20,7 @@
     "require": {
         "php": ">=7.0",
         "psr/http-message": "^1.0",
+        "psr/http-server-handler": "^1.0",
```

Comment:
The dependency here is `psr/http-server-middleware`

Reply:
💯 good catch!

Reply:
Fixed.